--- libavcodec/mlpdsp.c	2014-10-03 04:49:12.000000000 +0300
+++ libavcodec/mlpdsp-m.c	2014-10-08 12:24:00.000000000 +0300
@@ -131,8 +131,6 @@
     c->mlp_rematrix_channel = ff_mlp_rematrix_channel;
     c->mlp_select_pack_output = mlp_select_pack_output;
     c->mlp_pack_output = ff_mlp_pack_output;
-    if (ARCH_ARM)
-        ff_mlpdsp_init_arm(c);
     if (ARCH_X86)
         ff_mlpdsp_init_x86(c);
 }
